### ISSUE TITLE
[QUESTION] Delete function is not implemented in milvus v2.0.0-rc6

### ISSUE THREAD

**Issue by Nathansong** [milvus-io/pymilvus#725]
**What is your question?**
I try the v2.0.0-rc5 and v2.0.0-rc6, the first one will raise the information like : doesn't have delete methods for entities,
and the second raise Delete function is not implemented.  I wonder is something wrong here ?

**Comment by XuanYang-cn** [comment]
@Nathansong No, the behaviour is correct. We haven't implemented delete on the server side yet. We just added an API on SDK side and marked it as not-implemented.

Delete will be available on official 2.0.0, not on rc versions

**Comment by XuanYang-cn** [comment]
Closing for stale, delete is released long time ago
